doc/src/modules/physics/vector/advanced.rst

```diff
@@ -8,6 +8,44 @@ the features that will be implemented in the future will also be covered, along
 with unanswered questions about proper functionality. Also, common problems
 will be discussed, along with some solutions.
 
+Time Variable
+=============
+
+The objects in `sympy.physics.vector` and `sympy.physics.mechanics` rely on a
+default symbol for time. If symbols are created with ``dynamicsymbols()`` they
+are implicit functions of the default time varialb.e This default symbol can be
+accessed like so::
+
+   >>> from sympy.physics.vector import TIME
+
+or::
+
+   >>> from sympy.physics.mechanics import TIME
+
+The default behavior is::
+
+   >>> TIME
+   t
+   >>> from sympy.physics.vector import dynamicsymbols
+   >>> q = dynamicsymbols('q')
+   >>> q
+   q(t)
+   >>> q.diff(TIME)
+   Derivative(q(t), t)
+
+You can set a different symbol for time like so::
+
+   >>> from sympy import Symbol
+   >>> import sympy.physics.vector.functions
+   >>> tau = Symbol('tau')
+   >>> sympy.physics.vector.functions.TIME = tau
```

Comment:
This is incompatible with the suggested way to import `TIME` above

Reply:
I'm not sure what you are referring to above.